apps/docs/src/index.md

```diff
@@ -13,30 +13,17 @@ nav:
       position: 5
 ---
 
-<div class="mt-10 mx-10 mb-20">
-    <img src=".assets/shopware-frontends-logo.png" class="dark:hidden">
-    <img src=".assets/shopware-frontends-logo-dark.png" class="hidden dark:block">
-</div>
-
-<div class="text-center">
-
-**Shopware Composable Frontends** is Shopware's toolkit for creating **platform agnostic** custom storefronts. The demo store implementation is based on **Vue.js** and **Nuxt3**.
```

Comment:
I'd say this line was pretty important in explaining what is the project about. Do you consider placing it somewhere else?